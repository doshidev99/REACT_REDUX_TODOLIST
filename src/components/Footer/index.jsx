import React, { Fragment } from 'react';
import FilterLink from '../../containers/FilterLink';

export default () => (
  <Fragment>
    <FilterLink filter="SHOW_ALL">All</FilterLink>
    <FilterLink filter="SHOW_ACTIVE">Active</FilterLink>
    <FilterLink filter="SHOW_COMPLETED">Completed</FilterLink>
  </Fragment>
);
