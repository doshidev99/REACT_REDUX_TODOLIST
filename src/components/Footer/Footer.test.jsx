import React from 'react';
import { shallow } from 'enzyme';
import Footer from './index';

describe('Footer', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Footer debug />);

    expect(component).toMatchSnapshot();
  });
});
