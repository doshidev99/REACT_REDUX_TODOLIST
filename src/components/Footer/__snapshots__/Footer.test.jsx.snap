// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly in "debug" mode 1`] = `
<Fragment>
  <Connect(Link)
    filter="SHOW_ALL"
  >
    All
  </Connect(Link)>
  <Connect(Link)
    filter="SHOW_ACTIVE"
  >
    Active
  </Connect(Link)>
  <Connect(Link)
    filter="SHOW_COMPLETED"
  >
    Completed
  </Connect(Link)>
</Fragment>
`;
